wdn/templates_4.0/scripts/navigation.js

```diff
@@ -44,6 +45,72 @@ define(['jquery', 'wdn', 'modernizr', 'require'], function($, WDN, Modernizr, re
 		return setTimeout(callback, 0);
 	};
 
+	var _now = Date.now || function() {
+		return new Date().getTime();
+	};
+
+	var throttle = function(func, wait, options) {
+		var context, args, result;
+		var timeout = null;
+		var previous = 0;
+		if (!options) options = {};
+		var later = function() {
```

Comment:
Good point. These are pulled directly from vendor code. I will add a comment to that extent.